chart/jetstack-secure-gcm/charts/preflight/templates/_helpers.tpl

```diff
@@ -48,6 +48,7 @@ Selector labels
 {{- define "preflight.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "preflight.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
+app.kubernetes.io/component: preflight
```

Comment:
nit. Are we calling it preflight, or are we calling it `jetstack-secure-agent`, or somthing else? 
I guess preflight makes sense until we change the name of the chart.

Reply:
I am quite confused by this naming unknown: in places, I call it the "Jetstack Secure Platform agent", and sometimes I default to "the preflight agent".

@charlieegan3 thoughts?